.gitattributes

```diff
@@ -0,0 +1,7 @@
+###############################################################################
+# Set default behavior to automatically normalize line endings.
+###############################################################################
+* text eol=lf
```

Comment:
```suggestion
* -text
```

Reply:
This policy states that you don't mind a mix of line endings. Within a file, line endings could be mixed (and you can blame the editor if you want), but certainly newly introduced files may be created with CRLF line endings. This policy that you propose should mean you're OK with that. 
If you want to see LF line endings instead, we should state that.